--- Makefile.am~	2003-11-11 10:37:30.000000000 +0100
+++ Makefile.am	2003-12-29 11:52:20.264607184 +0100
@@ -16,6 +16,7 @@
 INC = @CPPFLAGS@
 LIB = @LIBS@
 AM_CFLAGS = @CFLAGS@
+LIBTOOL = @LIBTOOL@ --tag CC
 
 #   the default target
 lib_LTLIBRARIES = mod_clamav.la
--- Makefile.am.save	2005-12-26 21:21:33.000000000 +0200
+++ Makefile.am	2005-12-26 21:21:33.000000000 +0200
@@ -27,7 +27,7 @@
 MODULESOURCES = mod_clamav.c
 
 mod_clamav_la_SOURCES = $(MODULESOURCES)
-mod_clamav_la_LDFLAGS = -module `${APXS} -q LDFLAGS_SHLIB`
+mod_clamav_la_LDFLAGS = -module -avoid-version `${APXS} -q LDFLAGS_SHLIB`
 mod_clamav_la_CPPFLAGS = -I`${APXS} -q INCLUDEDIR`
 mod_clamav_la_CFLAGS = `${APXS} -q CFLAGS_SHLIB`
 CC = `${APXS} -q CC`
